src/Apps/ArtAdvisor/03-Instruction-Experiments/App.tsx

```diff
@@ -62,17 +64,17 @@ export const App: FC = () => {
         )
       })}
       <Spacer y={2} />
-      <Flex>
-        <Input
-          name="userInput"
-          placeholder="Enter your message here"
-          onChange={e => setUserInput(e.target.value)}
-        />
-        <Spacer x={4} />
-        <Button type="submit" onClick={onSubmit}>
-          Send
-        </Button>
-      </Flex>
+      <form onSubmit={onSubmit}>
+        <Flex>
+          <Input
```

Comment:
Wrapping input elements in a form ensures that "[implicit submission](https://www.tjvantoll.com/2013/01/01/enter-should-submit-forms-stop-messing-with-that/)" works —  i.e. being able to just hit enter on a simple form to submit it, which is desirable for our simple chat UI (could change if we eventually support multiline input).